src/App.vue

```diff
@@ -1,5 +1,10 @@
 <template>
   <div id="app">
+    <ToggleButton
+      :active="showDarkMode"
+      :bg="true"
+      @on-click-toggle-button="setToggleDarkMode"
```

Comment:
Vc pode chamar o `toggleDarkMode` direto aqui.
```suggestion
      @on-click-toggle-button="toggleDarkMode"
```